benches/generate_and_query/main.rs

```diff
@@ -49,6 +52,7 @@ impl Default for Arguments {
             num_threads: 10,
             num_points: 1_000_000,
             batch_size: 5000,
+            seed: 80_293_751_232,
```

Comment:
I changed the seed to `80_293_751_234`, since it wasn't behaving as nicely as this one, but you can play around a bit, if you want...